### PR TITLE
Update to Drafter 3.0.0 and Release 1.4.0

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

**Comment by w-vi** [comment]
@kylef  feel free to merge

File: src/parse_async.cc [hunk 1]
```diff
@@ -105,8 +105,7 @@ void AsyncParse(uv_work_t* request) {
     Baton* baton = static_cast<Baton*>(request->data);
 
     // Parse the source data
-    drafter::ParseBlueprint(baton->sourceData, baton->options | snowcrash::ExportSourcemapOption, baton->parseResult);
-
+    snowcrash::parse(baton->sourceData, baton->options | snowcrash::ExportSourcemapOption, baton->parseResult);
```
Comment (pksunkara) [inline src/parse_async.cc]: What? NO. This doesn't do the expansion and rendering of Refract.

**Comment by pksunkara** [comment]
@kylef Please fix.

File: src/parse_async.cc (hunk 1 above)
Comment (w-vi) [inline src/parse_async.cc]: https://github.com/apiaryio/drafter/blob/v2.3.1/src/drafter.cc#L20

Comment (pksunkara) [inline src/parse_async.cc]: Ah, I thought we changed the way it works. Never mind me then.

**Comment by pksunkara** [comment]
Since https://github.com/apiaryio/drafter/pull/356 is merged, please update submodule and ping here. Thanks.

**Comment by pksunkara** [comment]
👍

**Comment by pksunkara** [comment]
@kylef Please tag and release.

**Pull request merged**